pkg/network/setup/netpod/netpod.go

```diff
@@ -564,6 +568,32 @@ func (n NetPod) lookupMasquradeBridge(desiredIfacesSpec []nmstate.Interface) *nm
 	return nil
 }
 
+func calcQueuesCountByIfaceName(desiredQueueCount int,
+	ifaces []v1.Interface,
+	ifaceStatuses []v1.VirtualMachineInstanceNetworkInterface) map[string]int {
+
+	ifaceStatusesInDomainByName := vmispec.IndexInterfaceStatusByName(ifaceStatuses,
+		func(ifaceStatus v1.VirtualMachineInstanceNetworkInterface) bool {
+			return vmispec.ContainsInfoSource(ifaceStatus.InfoSource, vmispec.InfoSourceDomain)
+		})
+
+	queuesCountByIfaceName := map[string]int{}
+	for _, iface := range ifaces {
+		if iface.SRIOV != nil {
+			continue
+		}
+
+		ifaceStatus, inDomain := ifaceStatusesInDomainByName[iface.Name]
```

Comment:
What does `inDomain` mean? That the domain is already defined, and are we dealing with an existing VM?

Reply:
> That the domain is already defined, and are we dealing with an existing VM?

Exactly.
The network setup is uniform for the following three scenarios:
- Initial VM setup
- Setup for a migration target
- NIC hotplug / unplug when the VM is running.

Renamed `inDomain` to `existsInDomain`.